### ISSUE TITLE
BioJS demo page doesn't work

### ISSUE THREAD

**Issue by armish** [PathwayCommons/pcviz#14]
From A. Mark Roberts:

```
The demos for this package on the BioJS registry aren't rendering correctly, I think. 

http://biojs.io/d/biojs-pcviz

I'm seeing all blank DIVs. Maybe it's been that way for a while or maybe this package isn't maintained anymore.
```

To replicate this, visit either http://biojs.io/d/biojs-pcviz or http://workmen.biojs.net/demo/biojs-pcviz/simple

and this is what you get on the JavaScript Console:
<img width="1030" alt="screen shot 2015-08-06 at 1 32 03 pm" src="https://cloud.githubusercontent.com/assets/7809/9118608/927c4698-3c3f-11e5-95ac-6f00b6db5809.png">

**Comment by armish** [comment]
Just had a chance to take another look at this and realized that this is due to our policy with pathwaycommons server. See the response and the `X-Frame-Options` coming from our server:

<img width="1273" alt="screen shot 2015-08-25 at 3 15 33 pm" src="https://cloud.githubusercontent.com/assets/7809/9476799/b2a7587a-4b3c-11e5-8b77-d2ea70bcbf25.png">

We probably need to let our sys-admins know and remove this restriction (cc: @onursumer @emekdemir) or find a way to over-ride the default headers within PCViz.

**Comment by armish** [comment]
the code changes addressed by changes 0cb90da have not resolved this issue. This requires a sysadmin work on the server side.

**Comment by armish** [comment]
closed by: https://github.com/PathwayCommons/biojs-pcviz/commit/38aebb9a5d3b33e2b34aecea16d0fe39625636d1
